webviews/src/views/RokuAutomationView/AutoRunsEditor.svelte

```diff
@@ -0,0 +1,395 @@
+<script>
+    import { Trash, Copy, TriangleDown } from 'svelte-codicons';
+    import { dropzone, draggable } from './dnd';
+
+    export let runs;
+    export let selectedRun;
+
+    let runTable;
+    let runNameInput;
+    let runNameDialog;
+    let runNameDialogValue;
+    let confirmDialog;
+    let showContent = false;
+
+    const toggleDropDown = () => {
```

Comment:
Let's convert all of these to normal named functions. I.e.

`function toggleDropdown() { 
    ....
```

Reply:
yep